packages/slice-machine/components/ScreenshotChangesModal/DropZone.tsx

```diff
@@ -0,0 +1,97 @@
+import { ReactNode, useState } from "react";
+import { Flex } from "theme-ui";
+import { acceptedImagesTypes } from "@lib/consts";
+import { ToasterType } from "@src/modules/toaster";
+import useSliceMachineActions from "@src/modules/useSliceMachineActions";
+
+const sharedFlexSx = {
+  justifyContent: "center",
+  alignItems: "center",
+  height: "100%",
+  width: "100%",
+};
+
+interface DropZoneProps {
+  isVisible?: boolean;
+  onHandleDrop: (file: File) => void;
+  imageTypes?: string[];
+  children: ReactNode | ((options: { isDragActive: boolean }) => ReactNode);
+}
+
+const DropZone: React.FC<DropZoneProps> = ({
+  children,
+  onHandleDrop,
+  isVisible = false,
+  imageTypes = acceptedImagesTypes,
+}) => {
+  const { openToaster } = useSliceMachineActions();
+  const [isDragActive, setIsDragActive] = useState(false);
+
+  const handleDrop = (event: React.DragEvent<HTMLInputElement>) => {
+    event.preventDefault();
+    setIsDragActive(false);
+    const maybeFile = event.dataTransfer.files?.[0];
+    if (maybeFile) {
+      if (imageTypes.find((t) => `image/${t}` === maybeFile.type)) {
+        if (maybeFile.size > 128000000) {
+          return openToaster(
+            "File is too big. Max file size: 128Mo.",
```

Comment:
```suggestion
            "File is too big. Max file size: 128Mb.",
```
Is it supposed to be Mb?